examples/baseprofile.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: security-profiles-operator.x-k8s.io/v1alpha1
+kind: SeccompProfile
+metadata:
+  name: base
```

Comment:
Do you think we need some kind of description field for a profile, too? This way we could still name it `base`, but explain for which runtime (runc) and version it has been built and for which purpose.

Reply:
Maybe we could just include it in the name, combined with `targetWorkload`? e.g. `name: runc-v1.0.0-rc92` `targetWorkload: base`

Reply:
Yes this would also work from my perspective :+1:

Reply:
done